cmd/create/cluster/cmd.go

```diff
@@ -493,13 +517,53 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	// Autoscaling
+	autoscaling := args.autoscalingEnabled
+	if !autoscaling && interactive.Enabled() {
+		autoscaling, err = interactive.GetBool(interactive.Input{
+			Question: "Enable autoscaling",
+			Help:     cmd.Flags().Lookup("enable-autoscaling").Usage,
+			Default:  autoscaling,
+			Required: false,
+		})
+		if err != nil {
+			reporter.Errorf("Expected a valid value for enable-autoscaling: %s", err)
+			os.Exit(1)
+		}
+	}
+
+	minReplicas := args.minReplicas
+	maxReplicas := args.maxReplicas
+	if autoscaling && interactive.Enabled() {
+		minReplicas, err = interactive.GetInt(interactive.Input{
+			Question: "Min replicas",
+			Help:     cmd.Flags().Lookup("min-replicas").Usage,
+			Default:  minReplicas,
+			Required: true,
+		})
+		if err != nil {
+			reporter.Errorf("Expected a valid number of min replicas: %s", err)
+			os.Exit(1)
+		}
```

Comment:
Since `0` is not a valid value, you should check
```go
if minReplicas <= 0 {
  reporter.Errorf(...)
  os.Exit(1)
}
```
Better to exit early rather than let the user go through the entire flow and still get an API failure

Reply:
Fixed